CurrencyAPI/CurrencyAPI/Program.cs

```diff
@@ -1,17 +1,82 @@
+using System.Text.Json;
+
 namespace CurrencyAPI
 {
+    public class Bpi
+    {
+        public USD USD { get; set; }
+        public GBP GBP { get; set; }
+        public EUR EUR { get; set; }
+    }
+
+    public class EUR
+    {
+        public string code { get; set; }
+        public string symbol { get; set; }
+        public string rate { get; set; }
+        public string description { get; set; }
+        public double rate_float { get; set; }
+    }
+
+    public class GBP
+    {
+        public string code { get; set; }
+        public string symbol { get; set; }
+        public string rate { get; set; }
+        public string description { get; set; }
+        public double rate_float { get; set; }
+    }
+
+    public class Root
+    {
+        public Time time { get; set; }
+        public string disclaimer { get; set; }
+        public string chartName { get; set; }
+        public Bpi bpi { get; set; }
+    }
+
+    public class Time
+    {
+        public string updated { get; set; }
+        public DateTime updatedISO { get; set; }
+        public string updateduk { get; set; }
+    }
+
+    public class USD
+    {
+        public string code { get; set; }
+        public string symbol { get; set; }
+        public string rate { get; set; }
+        public string description { get; set; }
+        public double rate_float { get; set; }
+    }
     internal static class Program
     {
+        static HttpClient httpclient = new HttpClient();
+        static async Task Request()
```

Comment:
I suggest to move this code at the separate class. For example: BitcoinAPIController.